rdt-app/src/main/java/io/ona/rdt_app/interactor/PatientRegisterFragmentInteractor.java

```diff
@@ -125,7 +125,10 @@ private EventClient saveEventClient(JSONObject jsonForm, String encounterType, S
             dbClient = gson.fromJson(clientJson.toString(), org.smartregister.domain.db.Client.class);
         }
 
+        String providerId = RDTApplication.getInstance().getContext().userService().getAllSharedPreferences().fetchRegisteredANM();
```

Comment:
This can be simplified to `RDTApplication.getInstance().getContext().getAllSharedPreferences()`

Reply:
Done.